### PR TITLE
Stopped printing jwtSecret to command line on fail

### LINKED ISSUE #219: Make sure we don't print secrets on command line
Right now we're allowing the user to override particularly JWT_SECRET via the command line, and defaulting it to the JWT_SECRET env variable. This means that if the right args don't get passed, the secret will get printed to the command line 😬

### PR DESCRIPTION
**Pull request by AlexGilleran**
Fixes #219

### CODE REVIEW

File: magda-admin-api/src/index.ts [hunk 1]
```diff
@@ -50,13 +50,14 @@ const argv = yargs
         default: process.env.USER_ID || process.env.npm_package_config_userId
     })
     .option("jwtSecret", {
-        describe:
-            "Secret for decoding JWTs to determine if the caller is an admin",
+        // describe:
```
Comment (kring) [inline magda-admin-api/src/index.ts]: Did you mean to comment out these lines?

**Comment by kring** [comment]
Looks like you missed `magda-ckan-connector`.

**Comment by kring** [comment]
After these changes, it will still print the secrets to the console if you run with `--help`.  I'm not totally sure what the threat model is here so I can't really say if that's "as bad" a problem, but it seems less than ideal, at least.

I think it'd be better to `Object.assign`-in the secrets after yargs gets done with it rather than disabling help on failure.  That would prevent the secrets from being shown even when you run with `--help` and it would give us back the (useful) help when you run it incorrectly.

**Comment by AlexGilleran** [comment]
Assuming these changes don't end up 🐙 -ing...

Changed the whole approach so that there's a `addJwtSecretFromEnvVar` function that accepts an `argv` object, checks whether it's got a `jwtSecret` and if not adds one from env or package.json.

**Pull request merged**